usersync/cookie_test.go

```diff
@@ -359,7 +359,8 @@ func TestTrimCookiesClosestExpirationDates(t *testing.T) {
 	testCases := []aTest{
 		{maxCookieSize: 2000, expKeys: []string{"k1", "k2", "k3", "k4", "k5", "k6", "k7"}}, //1 don't trim, set
 		{maxCookieSize: 0, expKeys: []string{"k1", "k2", "k3", "k4", "k5", "k6", "k7"}},    //2 unlimited size: don't trim, set
-		{maxCookieSize: 800, expKeys: []string{"k1", "k5", "k4", "k3"}},                    //3 trim to size and set
+		// failing doing to size differences across platforms
```

Comment:
Is this still necessary?  Maybe check if this flakiness was addressed since we forked?

Reply:
@jorgeluisrocha did you check this ?

Reply:
yes! its still flaky.